### DISCUSSION TITLE
Congratulations!

### DISCUSSION THREAD

**Discussion by PachVerb** [community/community#13258] [Other Feature Feedback, Questions, & Ideas]
Congratulations, this feature is very useful. In general, github is fortunate to meet you, which has made me meet a lot of code masters. cheers!

**Comment by jfullstackdev** [comment]
other than that,

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
